dist/services/__test__/StructuralMetadataUtils.test.js

```diff
@@ -132,8 +124,24 @@ describe('StructuralMetadataUtils class', () => {
       const timespan = smu.findItem('123a-456b-789c-2d', structure);
       expect(timespan.end).toEqual('00:15:00.230');
     });
-    test('when end time exceeds (00:38:58.000) file duration (00:28:58.950)', () => {
+    test('when time is in mm:ss.ms (00:41.45) format', () => {
+      const timespan = smu.findItem('123a-456b-789c-1d', structure);
+      expect(timespan.begin).toEqual('00:00:41.450');
+    });
+    test('when time is in mm:ss.ms (10:41.45) format', () => {
+      const timespan = smu.findItem('123a-456b-789c-1d', structure);
+      expect(timespan.end).toEqual('00:10:41.450');
+    });
+    test('when time is in mm.ss (15.30) format', () => {
```

Comment:
I'm not sure if this is a valid value that needs to be supported.  Have you seen this in an old structure?

Reply:
No. I'll remove it :+1: